### PR TITLE
Macros: `PLAYER` -> `GET_PLAYER(globalCtx)`, `ACTIVE_CAM` -> `GET_ACTIVE_CAM(globalCtx)`

### PR DESCRIPTION
**Pull request by engineer124**
Even though this change is currently in https://github.com/zeldaret/oot/pull/908, I did end up deciding to make a separate PR because there are many actors I changed in https://github.com/zeldaret/oot/pull/908 dealing with `subCam`, and these two bulk macro changes from that PR make it harder to find and review the changes to subCam. So the purpose of this PR is to make sure those `subCam` changes aren't missed in review.

In the end, this PR changes the two Macros in the title, and fills in cases where the usage of this Macro was missed, as well as fill in new cases where these new macros applies (eg. `GET_PLAYER(camera->globalCtx)`, `GET_ACTIVE_CAM(camera->globalCtx)`)

### CODE REVIEW

**Review by Dragorn421 — CHANGES_REQUESTED** [review]
Could be added to actorfixer.py:
```py
    "PLAYER": "GET_PLAYER(globalCtx)",
    "ACTIVE_CAM": "GET_ACTIVE_CAM(globalCtx)",
```

And the formatter should be run

**Comment by fig02** [comment]
this is not necessarily a change I want to see right now, but something in the back of my mind: 
We know that these systems are designed to be portable, meaning they can work in any gamestate. Even if its not how things turned out in practice, it was definitely the intent. And with some modifications you could in theory have a new gamestate that contains an actor context and these camera variables. So would it be worth making the arg less specific than `globalCtx`?

If we decide not to, then another question I have is: should this macro make iteslf clear that it belongs to global context/play state? ie `PLAY_GET_ACTIVE_CAM` (or whatever global context ends up becoming)

Just food for thought, dont think we could make these kinds of decisions until our naming scheme for gamestates/play state is ironed out and implemented

**Comment by Roman971** [comment]
I think this should be good besides the stuff Dragorn mentioned in his review?

**Review by Dragorn421 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

File: include/macros.h [hunk 1]
```diff
@@ -22,9 +22,9 @@
 
 #define RGBA8(r, g, b, a) (((r & 0xFF) << 24) | ((g & 0xFF) << 16) | ((b & 0xFF) << 8) | ((a & 0xFF) << 0))
 
-#define PLAYER ((Player*)globalCtx->actorCtx.actorLists[ACTORCAT_PLAYER].head)
+#define GET_PLAYER(globalCtx) ((Player*)globalCtx->actorCtx.actorLists[ACTORCAT_PLAYER].head)
```
Comment (AngheloAlf) [inline include/macros.h:25]:
```suggestion
#define GET_PLAYER(globalCtx) ((Player*)(globalCtx)->actorCtx.actorLists[ACTORCAT_PLAYER].head)
```

File: include/macros.h [hunk 2]
```diff
@@ -22,9 +22,9 @@
...
 
-#define ACTIVE_CAM globalCtx->cameraPtrs[globalCtx->activeCamera]
+#define GET_ACTIVE_CAM(globalCtx) globalCtx->cameraPtrs[globalCtx->activeCamera]
```
Comment (AngheloAlf) [inline include/macros.h:27]:
```suggestion
#define GET_ACTIVE_CAM(globalCtx) ((globalCtx)->cameraPtrs[(globalCtx)->activeCamera])
```

**Comment by engineer124** [comment]
> Could be added to actorfixer.py:
> 
> ```python
>     "PLAYER": "GET_PLAYER(globalCtx)",
>     "ACTIVE_CAM": "GET_ACTIVE_CAM(globalCtx)",
> ```

@AngheloAlf Similar to your comment in MM, 

"This is a bad idea, because `GET_PLAYER(globalCtx)` contains the word `PLAYER`, so running this script a second time would produce `GET_GET_PLAYER(globalCtx)(globalCtx)` and so on"

That would also apply here. So should I take them out here? Any alternatives?

**Comment by fig02** [comment]
unrelated to the thing above, youll need to merge master and fix a small conflict

**Comment by AngheloAlf** [comment]
I suggest to remove it from actorfixer.py. It isn't a huge deal imo, and leaving it there would produce way more problems than it would solve.

**Comment by engineer124** [comment]
> I suggest to remove it from actorfixer.py. It isn't a huge deal imo, and leaving it there would produce way more problems than it would solve.

I agree

**Review by fig02 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Comment by Dragorn421** [comment]
I fixed the actorfixer script to only match whole-words so it won't replace PLAYER in GET_PLAYER. If you want to make sure try running the script with the PLAYER -> GET_PLAYER replace

**Comment by fig02** [comment]
in that case we can add this back to actor-fixer and merge this in after

**Review by Roman971 — APPROVED** [review]

**Pull request merged**